asydata/R3BasyDataLinkDef.h

```diff
@@ -0,0 +1,13 @@
+// 2017/04/07 12:02:26 hapol
```

Comment:
this file is excluded from the clang-format, for this you must add this line:
// clang-format off